include/seqan/platform.h

```diff
@@ -365,14 +381,14 @@ typedef int8_t __int8;     // nolint
 // ==========================================================================
 // C++ branch hints
 // ==========================================================================
-#if defined(COMPILER_GCC) || defined(COMPILER_CLANG) || defined(COMPILER_INTEL)
+#if !defined(STDLIB_VS)
```

Comment:
I would rather keep this as it was before, because it is a non-standard feature of these compilers. I know the effect is the same, but it is clearer if defined by compiler VS standard library (it has nothing to do with the standard library).